login/main.c

```diff
@@ -43,6 +44,11 @@ int main(int argc, char* argv[]) {
     return EXITCODE_PANIC;
   }
 
+  // Initialize syslog, if we're going to log.
+  if (config.options & SYSLOG) {
```

Comment:
Do we support setting this in the config file? Then it'd need to be later. (Which is fine, it was done in run before anyway.)

Reply:
We do, we have `disable-syslog`. I'll update the PR.